content/zh/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -136,8 +136,18 @@ with a standard set of labels. See [Well-Known Labels, Annotations and Taints](/
 -->
 ## 插曲：内置的节点标签 {#built-in-node-labels}
 
-除了你[添加](#attach-labels-to-node)的标签外，节点还预先填充了一组标准标签。
-参见[常用标签、注解和污点](/zh/docs/reference/labels-annotations-taints/)。
+除了你[添加](#step-one-attach-label-to-the-node)的标签外，节点还预制了一组标准标签。
```

Comment:
I dont know how we are dealing these inline heading cross-references, but it got updated anyway.